src/api/creatures.py

```diff
@@ -189,20 +194,23 @@ def feed_creature(user_id: int, creature_id: int, treat_sku: str):
                     change_in_affinity = (
                         5 if remaining_affinity >= 5 else remaining_affinity
                     )
+                    message = f"{stats["name"]} devoured the treat!"
 
                 elif treat_sku == stats["hated_treat"]:
                     change_in_happiness = -5 if remaining_happiness <= 95 else 0
                     change_in_affinity = -2 if remaining_affinity <= 98 else 0
                     change_in_hunger = 0
+                    message = f"{stats["name"]} spat out the treat!"
 
-                else:
-                    gold_earned = 3
-                    change_in_happiness = (
-                        2 if remaining_happiness >= 2 else remaining_happiness
-                    )
-                    change_in_affinity = (
-                        2 if remaining_affinity >= 2 else remaining_affinity
-                    )
+            else:
+                gold_earned = 3
+                change_in_happiness = (
+                    2 if remaining_happiness >= 2 else remaining_happiness
```

Comment:
I'll say it now since I realized; this would look cleaner if you wrote `min(remaining_happiness, 2)` instead. same with similar statements